src/fairchem/core/_cli_hydra.py

```diff
@@ -175,9 +177,8 @@ def __call__(self, dict_config: DictConfig) -> None:
             _set_deterministic_mode()
 
         self.runner: Runner = hydra.utils.instantiate(dict_config.runner)
-        self.runner.config = self.config
+        self.runner.initialize(self.config.job)
```

Comment:
@rayg1234 I am passing only the JobConfig. I dont think a runner needs the full config since it should be getting everything else upon instantiation, but let me know if you have thoughts